src/status_im/contexts/settings/wallet/keypairs_and_accounts/actions/view.cljs

```diff
@@ -0,0 +1,6 @@
+(ns status-im.contexts.settings.wallet.keypairs-and-accounts.actions.view
+  (:require [quo.core :as quo]))
+
+(defn view
```

Comment:
Yeah this code is mainly a placeholder for other upcoming work. So eventually this component won't only take props, it will also render some action buttons.

Reply:
We have it for actions like rename, delete keypair view and logic

Reply:
Sounds great 👍🏼